setup.cfg

```diff
@@ -1,10 +1,9 @@
 [versioneer]
 VCS = git
 style = pep440
-versionfile_source = version/_version.py
-versionfile_build = version/_version.py
+versionfile_source = src/_version.py
+versionfile_build = src/_version.py
```

Comment:
Probably we will need to adjust this to the template that @ascartabelli has been developing. If we end up to merge this PR after his, it is worth adding placing here a placeholder

Reply:
My PR is merged so you can add placeholders as you see fit.